### PR TITLE
test: Update httpd to the latest 2.4.62

### PR DESCRIPTION
**Pull request by jajik**
draft until Windows distribution becomes available

### CODE REVIEW

**Comment by jajik** [comment]
close #253

File: test/pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jboss.mod_cluster</groupId>
         <artifactId>mod_cluster-parent</artifactId>
-        <version>2.0.5.Final-SNAPSHOT</version>
+        <version>2.1.0.Final-SNAPSHOT</version>
```
Comment (rhusar) [inline test/pom.xml:7]: We will need to figure this out at some point :-) but at least we are testing the latest and greatest.

**Review by rhusar — APPROVED** [review]
LGTM

**Comment by rhusar** [comment]
Note sure why CI passed but failed in main - anyway here's the fix https://github.com/modcluster/mod_proxy_cluster/pull/256

**Pull request merged**